lib/Tunnel.js

```diff
@@ -26,8 +26,8 @@ Tunnel.prototype._init = function(cb) {
     var opt = self._opt;
 
     var params = {
-        path: '/',
-        json: true
+        //path: '/',
```

Comment:
No need to keep dead unused code/options around.

Reply:
fixed via d06a053